### PR TITLE
Update project setup instructions

### PR DESCRIPTION
**Pull request by oliverjam**
In London (for FAC16) I dropped my custom project boilerplate and went with Create React App. This was way smoother and let the cohort focus on React instead of annoying bundler issues. It's also better to get used to using a proper battle-tested setup (my boilerplate was never intended for   client projects but lots of people used it anyway 😱)

- Remove outdated requirements
- Add Create React App instructions
- Add Netlify instructions
- Remove link to outdated boilerplate repo

### CODE REVIEW

File: project.md [hunk 1]
```diff
@@ -32,8 +27,22 @@ Try to:
 
 ### Setup
 
-We'll go through the project setup together. All the information is in [this repo](https://github.com/oliverjam/minimal-react-setup)
+We're using [Create React App](https://create-react-app.dev/docs/getting-started) to handle all our bundling, linting etc.
+
+1. `npx create-react-app my-app-name` generate the project (might take a while)
```
Comment (astroash) [inline project.md]: can we replace `my-app-name` with `[my-app-name]` to show it's meant to be replaced?

**Review by bobbysebolao — APPROVED** [review]

**Review by astroash — APPROVED** [review]
Tiny suggestion.

Feel free to merge as is

**Comment by astroash** [comment]
@shiryz & @m4v15 for reference

File: project.md (hunk 1 above)
Comment (oliverjam) [reply project.md]:
```suggestion
1. `npx create-react-app [my-app-name]` generate the project (might take a while)
```
Sure

File: project.md [hunk 2]
```diff
@@ -32,8 +27,22 @@ Try to:
 
 ### Setup
 
-We'll go through the project setup together. All the information is in [this repo](https://github.com/oliverjam/minimal-react-setup)
+We're using [Create React App](https://create-react-app.dev/docs/getting-started) to handle all our bundling, linting etc.
+
+1. `npx create-react-app [my-app-name]` generate the project (might take a while)
+1. `cd my-app-name`
```
Comment (astroash) [inline project.md:33]: `[my-app-name]` 🙏🏽

**Pull request merged**